### ISSUE TITLE
Duplicate code in function instantaneous_rate

### ISSUE THREAD

**Issue by dholstein** [NeuralEnsemble/elephant#44]
@rmeyes  In function instantaneous_rate in module statistics.py:
if acausal: do A ; else: do B.  But seems A==B, so why ask for 'acausal' there?

**Comment by etorre** [comment]
Hi Richard. I checked and you are right, it seems redundant. Maybe the original implementation meant something else?
I also noticed that at the beginning of the routine, if sigma == 'auto', the sskernel() routine is summoned with bootstrap=True. But the confidence intervals calculated for bootstrap=True are not used, so we should set it to False. This makes the calculation faster.
        kernel_width = sskernel(spiketrain.magnitude, tin=None,
                                bootstrap=True)['optw']  -> bootstrap=False
